client/public/.well-known/walletconnec.txt

```diff
@@ -0,0 +1 @@
+33847eab-0224-4ee6-85b3-0fc80b4e5600=abfcd75a302f9096cd4fb41adfbe0148d7ef5172805da4ec1955ae46525fe0cc
```

Comment:
filename is incorrect -> change to `walletconnect.txt`

Reply:
Done.